### PR TITLE
Add two nao motion files

### PR DESCRIPTION
**Pull request by BenjaminDeleze**
ditto

### CODE REVIEW

File: projects/robots/softbank/nao/controllers/nao_demo/nao_demo.c [hunk 1]
```diff
@@ -58,7 +58,8 @@ static WbDeviceTag RShoulderPitch;
 static WbDeviceTag LShoulderPitch;
 
 // motion file handles
-static WbMotionRef hand_wave, forwards, backwards, side_step_left, side_step_right, turn_left_60, turn_right_60;
+static WbMotionRef hand_wave, forwards, backwards, side_step_left, side_step_right, turn_left_60, turn_right_60, taichi,
+  wipeForehead;
```
Comment (omichel) [inline projects/robots/softbank/nao/controllers/nao_demo/nao_demo.c:62]: Coding style:
```suggestion
  wipe_forehead;
```

File: projects/robots/softbank/nao/controllers/nao_demo_python/nao_demo_python.py [hunk 2]
```diff
@@ -195,6 +197,8 @@ def printHelp(self):
         print('[PageUp][PageDown]: open/close hands')
         print('[7][8][9]: change all leds RGB color')
         print('[0]: turn all leds off')
+        print('[T]: perform a taichi move')
+        print('[w]: wipe its forehead')
```
Comment (omichel) [inline projects/robots/softbank/nao/controllers/nao_demo_python/nao_demo_python.py:201]:
```suggestion
        print('[W]: wipe its forehead')
```

File: projects/robots/softbank/nao/controllers/nao_demo_python/nao_demo_python.py [hunk 3]
```diff
@@ -195,6 +197,8 @@ def printHelp(self):
         print('[PageUp][PageDown]: open/close hands')
         print('[7][8][9]: change all leds RGB color')
         print('[0]: turn all leds off')
+        print('[T]: perform a taichi move')
```
Comment (omichel) [inline projects/robots/softbank/nao/controllers/nao_demo_python/nao_demo_python.py:200]:
```suggestion
        print('[T]: perform Tai chi movements')
```

**Review by omichel — CHANGES_REQUESTED** [review]
Tai chi spelling is in two words, consequently, I would rename `Taichi.motion` to `TaiChi.motion` and fix the spelling elsewhere.

**Comment by BenjaminDeleze** [comment]
I also fixed some issues in the NAO controllers

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**